### PR TITLE
Cypress test for execution environments

### PR DESCRIPTION
**Pull request by ShaiahWren**
Issue: https://issues.redhat.com/browse/AAH-1001

This pr tests Execution Environments - list view, detail view, and readme file. 

As it is the tests are fairly simple. Lmk what other details you would like me to include.

### CODE REVIEW

File: test/cypress/integration/execution_environments.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      include_tags: 'latest',
+    });
+  });
+
+  beforeEach(() => {
+    cy.login(adminUsername, adminPassword);
+    cy.menuGo('Execution Environments > Execution Environments');
+  });
+
+  it.only('checks the EE list view', () => {
```
Comment (himdel) [inline test/cypress/integration/execution_environments.js:22]: `it.only` means other tests are ignored

File: test/cypress/integration/execution_environments.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+describe('execution environments', () => {
+  let adminUsername = Cypress.env('username');
+  let adminPassword = Cypress.env('password');
+  let num = (~~(Math.random() * 1000000)).toString();
```
Comment (himdel) [inline test/cypress/integration/execution_environments.js:4]: https://github.com/ansible/ansible-hub-ui/pull/1141#discussion_r739608451

**Review by himdel — APPROVED** [review]
LGTM standing on its own :+1: 

There's some code duplication with the functions added both here and in #1141 , but we can deal with that separately once both are merged, and move them to helpers :)

**Comment by ShaiahWren** [comment]
Finally closing this pr. :)

**Pull request merged**